scotty-resource.cabal

```diff
@@ -18,19 +18,28 @@ build-type:          Simple
 -- extra-source-files:  
 cabal-version:       >=1.10
 
+tested-with:         GHC == 8.10.7
+                   , GHC == 9.0.2
+                   , GHC == 9.2.8
+                   , GHC == 9.4.6
+                   , GHC == 9.6.4
+                   , GHC == 9.8.2
+                   , GHC == 9.10.3
+
 library
   exposed-modules:     
     Web.Scotty.Resource.Trans
   -- other-modules:       
   other-extensions:    
     OverloadedStrings
   build-depends:
-    base         >= 4.11     && < 4.12,
-    containers   >= 0.1.0.0  && < 0.6,
+    base         >= 4.11     && < 4.21,
+    containers   >= 0.1.0.0  && < 0.8,
     http-types   >= 0.8.2    && < 0.13,
-    scotty       >= 0.10     && < 0.12,
-    text         >= 0.11.3.1 && < 1.3,
-    transformers >= 0.3.0.0  && < 0.6,
+    scotty       >= 0.10     && < 0.23,
+    text         >= 0.11.3.1 && < 2.2,
+    unliftio     >= 0.2,
```

Comment:
Is there any reason not to have an upper bound on unliftio?

Reply:
Just copied from Scotty.